crates/forge_analyzer/src/definitions.rs

```diff
@@ -3299,6 +3305,23 @@ impl Visit for GlobalCollector<'_> {
     }
 }
 
+struct StringCollector {
+    strings: Vec<Atom>,
+}
+
+impl Visit for StringCollector {
+    fn visit_str(&mut self, n: &Str) {
```

Comment:
Should this handle templates as well?

Reply:
Fixed this, and made the vec String instead of Atom